owtf/settings.py

```diff
@@ -93,12 +93,12 @@
 TEMPLATES = os.path.join(OWTF_CONF, "build")
 STATIC_ROOT = os.path.join(OWTF_CONF, "build")
 
-# SMTP
-EMAIL_FROM = "you@your_server.com"
-SMTP_LOGIN = "login@your_server.com"
-SMTP_PASS = "your_password"
-SMTP_HOST = "your_mail_server.com"
-SMTP_PORT = 25
+# SMTP (Make changes here to setup SMTP server of your choice)
+EMAIL_FROM = "owtf.org@gmail.com"
+SMTP_LOGIN = "owtf-admin@owasp.org"
```

Comment:
you should not check in your credentials in git!

Reply:
Removed.